install.sh

```diff
@@ -687,13 +715,19 @@ main() {
     _variables && _setup_arguments "${@}"
 
     _check_existing_command() {
-        if command -v "${COMMAND_NAME}" 2>| /dev/null 1>&2; then
-            if grep -q COMMAND_NAME "${INFO_PATH}"/google-drive-upload.info 2>| /dev/null 1>&2; then
+        if COMMAND_PATH="$(command -v "${COMMAND_NAME}")"; then
+            if [ -f "${INFO_PATH}/google-drive-upload.info" ] && [ -f "${INFO_PATH}/google-drive-upload.binpath" ] && [ -f "${INFO_PATH}/google-drive-upload.configpath" ]; then
+                OLD_INSTALLATION_PRESENT="true" && . "${INFO_PATH}/google-drive-upload.info"
```

Comment:
Codacy found an issue: [Can't follow non-constant source. Use a directive to specify location.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6201392)